R/module_nested_tabs.R

```diff
@@ -359,7 +359,9 @@ srv_teal_module.teal_module <- function(id,
 
 .resolve_module_datanames <- function(data, modules) {
   stopifnot("data_rv must be teal_data object." = inherits(data, "teal_data"))
-  if (is.null(modules$datanames) || identical(modules$datanames, "all")) {
+  if (length(datanames(data))) {
```

Comment:
I don't think this can be a first condition. If `datanames` are not null then other conditions will be ignored.